### PR TITLE
Add text on cnf key rotation

### LINKED ISSUE #57: Refresh token binding after attestation expiration
Currently section 5.2 states:
> The client MUST also use the same key that was present in the "cnf" claim of the client attestation that was used for client authentication when the refresh token was issued.

In the case that the original client attestation JWT has expired, what should happen? If it is acceptable for the client to re-use the same key in the "cnf" field across attestation JWTs, then the client can easily generate a new attestation JWT with the same "cnf" key. But if the client needs to use a fresh "cnf" key, then I can think of three options:

1. Define a flow which endorses the new "cnf" key with the old "cnf" key.
2. Define a field in the client attestation JWT that contains a unique identifier for the client instance, and use that identifier to bind the refresh token to the client instance.
3. Explicitly don't permit this in the specification, add text to the specification that makes it clear that the "cnf" key must stay the same across refreshed client attestations.

My preference would be for option 2.

### PR DESCRIPTION
**Pull request by cobward**
Closes #57 

## 📑 Description
This PR makes explicit an already implicit feature of the specification - that it is not possible to rotate the confirmation key.

### CODE REVIEW

File: draft-ietf-oauth-attestation-based-client-auth.md
Comment (paulbastian) [inline draft-ietf-oauth-attestation-based-client-auth.md:249]:
```suggestion
## Rotation of Client Instance Key

This specification does not provide a mechanism to rotate the Client Instance Key in the Client Attestation JWT's "cnf" claim. If the Client Instance needs to use a new Client Instance Key for any reason, then it MUST request a new Client Attestation JWT from its Client Backend.

```

**Review by paulbastian — APPROVED** [review]

**Review by peppelinux — APPROVED** [review]

**Review by tlodderstedt — APPROVED** [review]

**Comment by cobward** [comment]
@tlodderstedt @paulbastian Any reason this can't be merged?

**Comment by paulbastian** [comment]
Waiting for @tplooker approval

**Review by tplooker — APPROVED** [review]

**Pull request merged**
